src/excmds.ts

```diff
@@ -594,6 +594,18 @@ async function idFromIndex(index?: number): Promise<number> {
     }
 }
 
+/** Close all tabs but current */
+//#background
+export async function tabonly() {
+    let tabs = browser.tabs.query({pinned: false, active: false})
```

Comment:
I think you also want `currentWindow: true` in this object.
https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/tabs/query

Without it, it closes all unpinned tabs across all windows except those which are the 'selected' tab in each window, which isn't what I would expect as a user.

Reply:
Oh idd, I didn't even think about multiple FF windows as I olways use just one, will update asap